src/test/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/ServiceProviderCommentTest.java

```diff
@@ -0,0 +1,43 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.jupiter.api.Test;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class ServiceProviderCommentTest {
+
+    @Test
+    void toEdifactTest() {
+        var edifact = new ServiceProviderComment("Something").toEdifact();
+
+        assertThat(edifact).isEqualTo("FTX+SPC+++Something'");
+    }
+
+    @Test
+    void testFromString() {
+        var edifact = "FTX+SPC+++red blood cell seen, Note low platelets'";
+        var parsedFreeText = ServiceProviderComment.fromString("FTX+SPC+++red blood cell seen, Note low platelets");
+        assertThat(parsedFreeText.getServiceProviderComment()).isEqualTo("red blood cell seen, Note low platelets");
+        assertThat(parsedFreeText.toEdifact()).isEqualTo(edifact);
+        assertThatThrownBy(() -> ServiceProviderComment.fromString("wrong value")).isExactlyInstanceOf(IllegalArgumentException.class);
```

Comment:
This is a separate test so I'd prefer to see it in its own `@Test`

(Applies to other tests too.)

Reply:
Is it better? 66a1b711466ddb4a0c3a013b4ce5ac91ac98d643
Thanks!